### PR TITLE
Glossy Gorrilas

### PR DESCRIPTION
**Pull request by fliepeltje**

### CODE REVIEW

**Comment by SebastiaanZ** [comment]
Merge issues are preventing the linter from running in the CI. We'll have to remove a GitHub Actions file to prevent it from running on new PR/push events before merging this PR.

File: glossy-gorillas/core/models/market.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+class ListingStatus(models.TextChoices):
+    AVAILABLE = "A", _("Listing Available")
+    NEGOTIATING = "N", _("In Negotiations")
+    SCHEDULED = "S", _("Agreed and Scheduled")
+    FINALIZED = "F", _("Trade Finalized")
+
+
+class Listing(models.Model):
+    """A listing of a possesed good on the market
```
Comment (eivl) [inline glossy-gorillas/core/models/market.py:18]: nice docstrings and inline comments.

File: glossy-gorillas/git_basics.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# Overview
```
Comment (eivl) [inline glossy-gorillas/git_basics.md:1]: amazing that this is documented. I approve.

**Review by eivl — COMMENTED** [review]
Fantastic code quality, excellent documentation, and tests included.

File: .github/workflows/gorilla-lint.yml [hunk 3]
```diff
@@ -14,4 +14,4 @@ jobs:
           python -m pip install --upgrade pip
           pip install flake8
       - name: Lint with flake8
-        run: flake8 . --isolated --exclude=.cache,.venv,.svn,CVS,.bzr,.hg,.git,__pycache__,.tox --max-line-length=100
+        run: flake8 . --isolated --exclude=.cache,.venv,.svn,CVS,.bzr,.hg,.git,__pycache__,.tox --max-line-length=119
```
[the remark this answers is not in the document]
Comment (patryk-tech) [reply .github/workflows/gorilla-lint.yml:17]: Not my team, but worth noting... That is actually the Django standard, as models, fields, mixins, etc. can get quite long.

> An exception to PEP 8 is our rules on line lengths. Don’t limit lines of code to 79 characters if it means the code looks significantly uglier or is harder to read. We allow up to 119 characters as this is the width of GitHub code review; anything longer requires horizontal scrolling which makes review more difficult. This check is included when you run flake8. Documentation, comments, and docstrings should be wrapped at 79 characters, even though PEP 8 suggests 72.

https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/

[the remark this answers is not in the document]
Comment (SebastiaanZ) [reply .github/workflows/gorilla-lint.yml:17]: It's also the line length that we've set for this Code Jam.

**Comment by eivl** [comment]
Im getting poetry errors and overall errors all over the place when I try to run it. could I the team contribute to some screenshots?

**Comment by eivl** [comment]
https://i.imgur.com/zRAlpK5.png

this is what i look at. is it as intended?

**Comment by fliepeltje** [comment]
> https://i.imgur.com/zRAlpK5.png
> 
> this is what i look at. is it as intended?

Unfortunately that is correct; a bit of context:

The PR containing all the styling ended up not getting merged, so all we have is extremely basic styling, or more accurately: placeholder-type styling. This styling PR was authored in time, however it contained a bunch of merge conflicts. Hours before the deadline I ended up having computer problems and I had to resolve the conflicts on a 6 year old chromebook that had no coding tools installed yet. I was not able to fix the PR in time for the deadline and ended up sending a modmail to ask if we could make the merge retrospectively. After a while I got a response that this was possible, so I made the PR again, but it never got merged and the thread was closed. 

If you'd like I could provide screenshots of how it looks like with the styling PR.

**Pull request merged**